tools/webots.py

```diff
@@ -78,6 +92,12 @@ def exec_build(roles):
         cprint(f"unable to build code, exit code {exit_code}", "red", attrs=["bold"])
         sys.exit(exit_code)
 
+    print("Setting target 'generic'...")
```

Comment:
Why do you set the target to the passed target and then set the target to generic after clean, configure and build? A comment would be nice here if possible. Comments a nice here for people that don't understand... like me

Reply:
99% of the time we are compiling for target `generic`. So its probably best to set it back to it after building a docker image for the given target.

Reply:
```suggestion
    print("Setting back to target 'generic'...")
```